doc/files/opensc.conf.5.xml.in

```diff
@@ -1839,6 +1839,18 @@ app <replaceable>application</replaceable> {
 						See <xref linkend="card_drivers"/>
 				</para></listitem>
 			</varlistentry>
+			<varlistentry>
+                               <term>
+                                       <envar>OPENSSL_CONF</envar>
+                               </term>
+				<listitem><para>
+                                               Override the path to the OpenSC specific OpenSSL
+                                               configuration. This is useful to configure FIPS
+                                               mode and make OpenSC follow it. By default, OpenSC
+                                               tries to load <filename>@sysconfdir@/opensc/openssl.cnf</filename>
```

Comment:
Is it possible to add a reference to man 5 config, fips_config and x509v3_config for the definition of this file? Could also be relevant to the SEE ALSO section.